src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -1813,6 +1813,31 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
     Use the `average()` function to return the average value for an attribute. It takes a single attribute name as an argument. If a value of the attribute is not numeric, it will be ignored when aggregating. If data matching the query's conditions is not found, or there are no numeric values returned by the query, it will return a value of null.
   </Collapser>
 
+  <Collapser
```

Comment:
blob() should be in the evaluable section rather than the aggregator section.

Reply:
Could you move dimensions() and buckets() down as well?  Round() and mod() have aggregator versions, but it might make more sense to have them in the evaluable section as well, what do you think?

Reply:
Great catch, thank you!! Yes, I think round() and mod() are more helpful to have in the evaluable section. I'll do some shuffling of all the aforementioned functions

Reply:
Thank you!